MessagePack objects wrote force_f32 past the end of their allocation: zend_object
must be the last member (properties_table is a trailing array). Not fixed upstream.

--- msgpack-3.0.1.orig/msgpack_class.c
+++ msgpack-3.0.1/msgpack_class.c
@@ -10,8 +10,8 @@
 typedef struct {
     long php_only;
     zend_bool assoc;
-    zend_object object;
     zend_bool force_f32;
+    zend_object object;
 } php_msgpack_base_t;
 
 typedef struct {
